backend/mypy.ini

```diff
@@ -29,5 +28,35 @@ plugins =
     mypy_django_plugin.main,
     mypy_drf_plugin.main
 
+mypy_path = $MYPY_CONFIG_FILE_DIR
+
 [mypy.plugins.django-stubs]
 django_settings_module = decompme.settings
+
+[mypy-m2c.*]
```

Comment:
We shouldn't need settings blocks for either of these since we're not even scanning for them, right? (we shouldn't be running mypy on either of them if we are)

Reply:
dmypy doesn't support `follow-imports=false` so we do scan them. see the commit which originally added this